ui/components/component-library/text-field/text-field.js

```diff
@@ -75,7 +73,7 @@ TextField.propTypes = {
   /**
    * The props to pass to the clear button
    */
-  clearButtonProps: PropTypes.shape(Box.PropTypes),
+  clearButtonProps: PropTypes.shape(ButtonIcon.PropTypes),
   /**
    * TextField accepts all the props from TextFieldBase and Box
    */
```

Comment:
Missed this update, but these no longer need to be required because this can operate as an uncontrolled component. It is only when `showClearButton` is true that these are required